classlib/src/main/java/org/teavm/classlib/java/io/TPrintStream.java

```diff
@@ -301,30 +301,21 @@ private void printSB() {
         sb.setLength(0);
     }
 
-    private Appendable appendable;
-
-    private Appendable getAppendable() {
-        if (appendable == null) {
-            appendable = new Appendable() {
-                @Override
-                public Appendable append(CharSequence csq) {
-                    print(csq, 0, csq.length());
-                    return this;
-                }
-
-                @Override
-                public Appendable append(CharSequence csq, int start, int end) {
-                    print(csq, start, end);
-                    return this;
-                }
-
-                @Override
-                public Appendable append(char c) {
-                    print(c);
-                    return this;
-                }
-            };
-        }
-        return appendable;
+    @Override
+    public Appendable append(CharSequence csq) {
+        print(csq, 0, csq.length());
+        return this;
+    }
+
+    @Override
+    public Appendable append(CharSequence csq, int start, int end) {
+        print(csq, start, end);
```

Comment:
I guess you should change first parameter to something like this: `csq != null ? csq : "null"`. Otherwise tests fail.

Reply:
Right, my bad. Fixed this one.

BTW, how can I run a single TeaVM test? What should be the command line options for this? I don't want to run the full test set, but PrintStreamTest class, or even a single PrintStreamTest#append_null method.

Reply:
> BTW, how can I run a single TeaVM test?

Just like any other single JUnit test. For example [see this answer](https://stackoverflow.com/questions/22505533/how-to-run-only-one-unit-test-class-using-gradle). Or just get IDEA and click arrow button right behind the method line.

Reply:
I use NetBeans and it can run individual tests for Maven projects, but not for Gradle projects. At least I don't have luck with this IDE. Thank you for the link!